data/json/items/armor/suits_protection.json

```diff
@@ -5724,5 +5724,143 @@
       "msg": "The %s pipes stop humming.",
       "target": "armor_nomad_chainmail"
     }
+  },
+  {
+    "id": "mopp_jacket",
+    "type": "ARMOR",
+    "name": { "str": "JSLIST overcoat" },
+    "description": "A waist-length overcoat with an integral chemical hood, an expandable pocket sewn into the left sleeve, and Velcro fastenings on the wrists and zipper, this jacket forms half of the US military’s CBRN JSLIST overgarment. Constructed with an inner layer of carbon discs that keep harmful chemicals from penetrating through the suit and a selectively permeable membrane that lets perspiration escape, the garment’s meant to be either worn over a regular uniform or alone for maximum breathability. Unsealed and exposed to the elements, it will likely last a day before its carbon layer is exhausted and the PPE loses its protective nature.",
```

Comment:
I think the last sentence of this is just confusing if its environmental protection doesn't actually degrade after a day's exposure.

Reply:
It will, I just haven’t implemented the json for that yet. will be basically copying the functionality of the mi-go plates.